### PR TITLE
Install and Setup RSpec

### PR DESCRIPTION
**Pull request by mksndz**

### CODE REVIEW

**Comment by mksndz** [comment]
@cgalarza just a draft for now, i'll add some additional specs before seeking to merge

**Comment by mksndz** [comment]
lets :shipit:

File: db/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM mysql:5
```
Comment (cgalarza) [inline db/Dockerfile:1]: @mksndz Do we still need this?

Comment (mksndz) [reply db/Dockerfile]: nope, removed!

**Comment by cgalarza** [comment]
👍 to merge. Do you think we should tag versions?

**Comment by mksndz** [comment]
sure. `1.0.0`? i'm not experienced in version tagging

**Comment by cgalarza** [comment]
I'll tag the current version as 1.0.0, and when we release again we can up the version.

I'll also add a version file to the application, if that sounds good.

**Comment by mksndz** [comment]
sounds good. are we committing to [semantic versioning](https://semver.org/)? otherwise, what guides our version numbering?

**Comment by cgalarza** [comment]
Yea, I would like to follow semantic versioning. It's hard to retroactively do it specially when we are not familiar with the codebase. Because of that I am okay with start with tagging the code that is on master right now as v1.0.0 and then moving forward from there. Mostly, I want a marker so we know what the last stable release was in case we need to rollback.

**Pull request merged**